tests/endpoints/v1/test_events.py

```diff
@@ -60,17 +60,54 @@ def test_search_params_underscores(self, client):
                 "start_around.scale": "5d",
                 "place.scope": "place1,place2",
                 "place.exact": "place3",
-                "start.gte": "2016-03-01T00:00:00.000000",
-                "start.lt": "2016-04-01T23:59:59.999999",
+                "start.gte": "2016-03-01",
+                "start.lt": "2016-04-01",
                 "start.tz": "Pacific/Auckland",
-                "end.gte": "2016-05-01T00:00:00.000000",
-                "end.lt": "2016-06-01T23:59:59.999999",
+                "end.gte": "2016-05-01",
+                "end.lt": "2016-06-01",
                 "end.tz": "Pacific/Auckland",
-                "active.gte": "2016-03-01T00:00:00.000000",
-                "active.lt": "2016-04-01T23:59:59.999999",
+                "active.gte": "2016-03-01",
+                "active.lt": "2016-04-01",
                 "active.tz": "Pacific/Auckland",
-                "updated.gte": "2016-03-01T00:00:00.000000",
-                "updated.lt": "2016-04-01T23:59:59.999999",
+                "updated.gte": "2016-03-01",
+                "updated.lt": "2016-04-01",
+                "updated.tz": "Pacific/Auckland",
+            },
+            verify=True,
+        )
+
+    @with_mock_client()
+    def test_search_datetimerange_params(self, client):
+        client.events.search(
+            start__gte="2016-03-01",
+            start__lt="2016-04-01",
+            start__tz="Pacific/Auckland",
+            end__gte="2016-05-01 13:13:13",
```

Comment:
Any specific reason why you pass a non iso format in this test?

Reply:
Note: this applies to many places, including the use cases.

Reply:
No specific reason.

I used this format as it seemed to be the one used in the related issues's example.

I'll update it now 🙂